calendar-bundle/src/CalendarEventsGenerator.php

```diff
@@ -0,0 +1,287 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Contao\CalendarBundle;
```

Comment:
This class needs to be moved into a sub-namespace. Only the `ContaoCalendarBundle` class can be in the top namespace.